src/dawn/CodeGen/Cuda/MSCodeGen.cpp

```diff
@@ -161,6 +161,43 @@ void MSCodeGen::generateKCacheFillStatement(MemberFunction& cudaKernel,
                           "] =" + ss.str());
 }
 
+bool MSCodeGen::intervalPreviouslyAccessed(const int accessID, iir::Interval const& targetInterval,
```

Comment:
can we get 1 line of doc what this does to not have to read the method?

Reply:
done